src/latticpp/marshal/gohandle.h

```diff
@@ -177,20 +183,29 @@ namespace latticpp {
     using PublicKey = GoHandle<GoType::PublicKey>;
     using Plaintext = GoHandle<GoType::Plaintext>;
     using Ciphertext = GoHandle<GoType::Ciphertext>;
+    using CiphertextQP = GoHandle<GoType::CiphertextQP>;
     using RotationKeys = GoHandle<GoType::RotationKeys>;
     using SwitchingKey = GoHandle<GoType::SwitchingKey>;
     using CKGProtocol = GoHandle<GoType::CKGProtocol>;
+    using CKGCRP = GoHandle<GoType::CKGCRP>;
     using CKGShare = GoHandle<GoType::CKGShare>;
     using RKGProtocol = GoHandle<GoType::RKGProtocol>;
+    using RKGCRP = GoHandle<GoType::RKGCRP>;
     using RKGShare = GoHandle<GoType::RKGShare>;
     using CKSProtocol = GoHandle<GoType::CKSProtocol>;
     using CKSShare = GoHandle<GoType::CKSShare>;
     using RTGProtocol = GoHandle<GoType::RTGProtocol>;
+    using RTGCRP = GoHandle<GoType::RTGCRP>;
     using RTGShare = GoHandle<GoType::RTGShare>;
     using Ring = GoHandle<GoType::Ring>;
+    using RingQP = GoHandle<GoType::RingQP>;
     using Poly = GoHandle<GoType::Poly>;
+    using PolyQP = GoHandle<GoType::PolyQP>;
     using UniformSampler = GoHandle<GoType::UniformSampler>;
     using PRNG = GoHandle<GoType::PRNG>;
+    using MetaData = GoHandle<GoType::MetaData>;
+    using BasisExtender = GoHandle<GoType::BasisExtender>;
+
```

Comment:
Does `UniformSampler` need a synonym?

Reply:
Actually since `UniformSampler` is no longer used by the multi-key code, we can remove it altogether from the wrapper